File: src/main/java/de/schramm/royalbash/controller/CommitGameEventRequest.java
package de.schramm.royalbash.controller;

import de.schramm.royalbash.controller.service.gameevent.GameEvent;
import lombok.Builder;
import lombok.Value;

@Value
@Builder
class CommitGameEventRequest {

    private final String gameId;
    private final GameEvent event;
}

File: src/main/kotlin/de/schramm/royalbash/controller/CommitGameEventRequest.kt
package de.schramm.royalbash.controller

import de.schramm.royalbash.controller.service.gameevent.GameEvent
import de.schramm.royalbash.controller.service.gameevent.NoOpEvent

internal class CommitGameEventRequest(val event: GameEvent) {
    constructor(): this(event = NoOpEvent())
}
